Source/WebCore/dom/Document.h

```diff
@@ -2270,6 +2273,8 @@ class Document
 
     MediaProducerMediaStateFlags m_mediaState;
 
+    bool m_fireMutationEvents = true;
```

Comment:
Boolean variables need a prefix in WebKit. This should be `m_shouldFireMutationEvents` and the setter `setShouldFireMutationEvent()`. Your getter is already correct.

Reply:
> Boolean variables need a prefix in WebKit. This should be `m_shouldFireMutationEvents` and the setter `setShouldFireMutationEvent()`.

Thanks — made those changes